microsoft-365/enterprise/ms-cloud-germany-transition-phases.md

```diff
@@ -55,22 +55,36 @@ The following sections contain actions and effects for workloads as they progres
+| Subscriptions are transferred, and licenses are reassigned. | Corresponding Office 365 Global services subscriptions are purchased in the Office 365 Global instance for the transferred Microsoft Cloud Deutschland subscriptions. Users with assigned Microsoft Cloud Deutschland licenses will be assigned licenses in the Office 365 Global instance. Legacy Microsoft Cloud Deutschland subscriptions are removed from the Office 365 services tenant on completion.| <ul><li>Changes to existing subscriptions will be blocked (for example, no new subscription purchases or seat count changes) during this phase.</li><li>License assignment changes will be blocked.</li><li>The Microsoft Cloud Deutschland subscription will be migrated to corresponding Office 365 Global services subscription. The Office 365 Global services offer of that subscription is defined by Microsoft (also known as _Offer mapping_).</li><li>The number of features (service plans) offered by Office 365 services can be larger than in the original Microsoft Cloud Deutschland offer. User licenses in Office 365 services will be equivalently assigned to similar Microsoft Cloud Deutschland features (service plans). User licenses of all users will be automatically assigned to the new features. The admin needs to take an explicit action to disable those licenses, if needed. </li><li>When subscription migration is complete, both Office 365 services and Microsoft Cloud Deutschland  subscriptions will be visible in the Office 365 Admin Portal, with the status of Microsoft Cloud Deutschland subscriptions as _deprovisioned_. </li><li>Users will be reassigned licenses that are tied to the new Office 365 services subscriptions. Any customer processes that have dependencies on MCD subscriptions or SKU GUIDs will be broken and need to be revised with the Office 365 services offering. </li><li>New subscriptions in the Office 365 services will be purchased with the new term (monthly/quarterly/yearly), and the customer will receive a prorated refund for the unused balance of the Microsoft Cloud Deutschland subscription. </li><li>Partner Microsoft Cloud Deutschland tenants won't be migrated. CSP customers will be migrated to Office 365 services under the new Office 365 services tenant of the same partner. After customer migration, the partner can manage this customer only from the Office 365 services tenant. </li><li>Additional functionality is available (for example, Microsoft Planner and Microsoft Flow), unless disabled by tenant admin. For information about how to disable service plans that are assigned to users' licenses, see [Disable access to Microsoft 365 services while assigning user licenses](disable-access-to-services-while-assigning-user-licenses.md).</li></ul> |
 ||||
 
 ## SharePoint Online (Phase 4)
 
 **Applies to**: All customers using SharePoint Online
 
+In case you are still using SharePoint 2013 workflows, limit SharePoint 2013 workflows the use during the SharePoint Online migration.
```

Comment:
```suggestion
In case you are still using SharePoint 2013 workflows, limit the use of SharePoint 2013 workflows during the SharePoint Online migration.
```